Road/notebooks/data_cleaning.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            if text == "":  # check if cleaned text is empty
</pre><p>If the message is empty, it is preferable to check it first, then apply the function to the text if it is not empty.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>